src/components/Dashboard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Landing } from './Landing';
+import { ErrorMessage } from './ErrorMessage';
+
+
+export const Dashboard = (props) => {

```

Comment:
```suggestion
export const Dashboard = ({ errorMessage} ) => {
```

You can destructure props like this if you want to. can be useful if you're injecting a bunch of props in a function component, as it saves you IDE space instead of doing it just before you render